src/org/labkey/test/pages/core/login/AuthDialogBase.java

```diff
@@ -0,0 +1,97 @@
+package org.labkey.test.pages.core.login;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.ToggleButton;
+import org.labkey.test.params.login.AuthenticationProvider;
+import org.openqa.selenium.WebDriver;
+
+public abstract class AuthDialogBase<T extends AuthDialogBase> extends ModalDialog
+{
+    private final LoginConfigRow _row;
+
+    protected AuthDialogBase(AuthenticationProvider<org.labkey.test.pages.ldap.LdapConfigureDialog> provider, WebDriver driver)
```

Comment:
Remove reference to code in a different rep:
```suggestion
    protected AuthDialogBase(AuthenticationProvider provider, WebDriver driver)
```